checkov/common/sca/output.py

```diff
@@ -238,52 +250,44 @@ def _add_to_report_licenses_statuses(
 
 
 def add_to_reports_cves_and_packages(
```

Comment:
why not to create a new function with all the changes for dependency tree scan results and to leave the old one for twistlock? 
the old one faster and when we dont have any dependencies we dont really need to make all these steps to add a record.

Reply:
we wanted to keep the same flow for all the cases, @Saarett @ayajbara @matansha WDT?